cmd/storer2elastic/dataprocessor/dataReplayer.go

```diff
@@ -0,0 +1,578 @@
+package dataprocessor
```

Comment:
Warnings found in dataprocessor/disabled/disabledNodesShuffler.go L14
Also, please fix all weak warnings (and warnings) from this file

Another "hard to digest struct".

Reply:
done. re-activates all code inspection checks as they were disabled